server/src/main/java/org/candlepin/resource/JobResource.java

```diff
@@ -184,7 +191,8 @@ public SchedulerStatusDTO setSchedulerStatus(
         @QueryParam("origin") Set<String> origins,
         @QueryParam("executor") Set<String> executors,
         @QueryParam("after") @DateFormat Date after,
-        @QueryParam("before") @DateFormat Date before) {
+        @QueryParam("before") @DateFormat Date before,
+        @Context PageRequest pageRequest) {
```

Comment:
Does OpenAPI not allow for specifying a thing that comes from the context, nor specifying that an object's properties should be unboxed for documentation? Seems like a pretty major shortcoming in an already irritatingly limited "API".

Neither of those options sound particularly great to me, but of the two, pulling from the context makes the most sense, as we have to inject the page into the context later anyway, so it's more complete/consistent from an implementation standpoint.

Spec tests are probably unnecessary here, since we're testing the paging parameters on the frontend and backend, but I can see how easy it is to add a couple spec tests just for the sake of end-to-end verification. If it's not too painful, I'll throw in a couple tests.

Reply:
Also, yes, after this gets merged, the spec first branch is in need of a rebase. I know of at least three areas that have changed out from under it, but I'm unsure if the resources have been converted yet or not.

Reply:
Removed the context param and added spec tests